src/Microsoft.ApplicationInsights.AspNetCore/TelemetryInitializers/DomainNameRoleInstanceTelemetryInitializer.cs

```diff
@@ -28,6 +28,12 @@ protected override void OnInitializeTelemetry(HttpContext platformContext, Reque
                 var name = LazyInitializer.EnsureInitialized(ref this.roleInstanceName, this.GetMachineName);
                 telemetry.Context.Cloud.RoleInstance = name;
             }
+
```

Comment:
Do we still need `DomainNameRoleInstanceTelemetryInitializer` initializer if this name will be set by ApplicationInsights.dll anyway? You removed it from the `WindowsServer` package, why not remove it here?